deployments/peerscout/peerscout--stg.yaml

```diff
@@ -0,0 +1,69 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: peerscout--stg
+  namespace: peerscout
+  annotations:
+    fluxcd.io/automated: 'true'
+    filter.fluxcd.io/chart-image: glob:*
+spec:
+  releaseName: peerscout--stg
+  chart:
+    git: https://github.com/elifesciences/data-science-dags
+    path: charts/peerscout
+    ref: develop
+
+  values:
+    image:
+      repository: elifesciences/data-science-dags_peerscout-api_unstable
+      tag: 39aa9515d6bc752e246f5715c468f891135675dc
+    env:
+      DATA_SCIENCE_STATE_PATH: s3://staging-elife-data-pipeline/airflow-config/data-science/state
+      GOOGLE_APPLICATION_CREDENTIALS: /root/.gcp/credentials.json
+      DEPLOYMENT_ENV: staging
+      SPACY_LANGUAGE_MODEL_NAME: "en_core_web_lg"
+    volumeMounts:
+    - name: aws-credentials-sealed-secret-volume
+      mountPath: /root/.aws
+      readOnly: true
+    - name: gcp-credentials-sealed-secret-volume
+      mountPath: /root/.gcp
+      readOnly: true
+    volumes:
+    - name: aws-credentials-sealed-secret-volume
+      secret:
+        secretName: aws-credentials
+    - name: gcp-credentials-sealed-secret-volume
+      secret:
+        secretName: gcp-credentials
+    ingress:
+      enabled: "true"
+      hosts:
+      - host: peerscout--test.elifesciences.org # this is `--test`, not `--stg` because it was setup this way on the old cluster, and is an external service.
+        paths:
+        - "/"
+      annotations:
+        nginx.ingress.kubernetes.io/auth-type: basic
+        nginx.ingress.kubernetes.io/auth-secret: peerscout-http-basic-auth--stg
+        nginx.ingress.kubernetes.io/auth-realm: 'Authentication Required - peerscout'
+
+    resources:
+      requests:
```

Comment:
I assume that you've just done a straight copy with some patching where needed, but wonder if once deployed we need to also need to measure and set a limit?

Might also want to consider some analysis of CPU requirements.

Reply:
CPU requirements are so low  - excluding startup most of the time it's < 1m CPU usage, I thought the default would be fine (50m).

Memory stays static, at just under 1.7GB.

Peerscout prod is similar: occasionally uses spikes to 20m, memory usage static at just under 1.8GB

Reply:
I've pushed some request and limits in line with these